projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/disable-core-nux.js

```diff
@@ -5,10 +5,6 @@ import '@wordpress/nux'; //ensure nux store loads
 // Disable nux and welcome guide features from core.
 const unsubscribe = subscribe( () => {
 	dispatch( 'core/nux' ).disableTips();
-	if ( select( 'core/edit-post' )?.isFeatureActive( 'welcomeGuide' ) ) {
-		dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' );
```

Comment:
Prevent the disabling of the core welcome guide

Reply:
Should we also disable preventing the core welcome guide on `core/edit-site` since it's also the editor?

Reply:
Makes sense. The same changes on `core/edit-site` show this banner when you get to edit a template for the first time:

![image](https://github.com/user-attachments/assets/c5d1dcce-7dc3-4874-80a5-87565e94b057)